src/steamcompmgr.cpp

```diff
@@ -273,12 +273,13 @@ static Bool		drawDebugInfo = False;
 static Bool		debugEvents = False;
 static Bool		steamMode = False;
 static Bool		alwaysComposite = False;
-static Bool		takeScreenshot = False;
 static Bool		useXRes = True;
 
 std::mutex wayland_commit_lock;
 std::vector<ResListEntry_t> wayland_commit_queue;
 
+static std::atomic< bool > g_bTakeScreenshot = false;
```

Comment:
Fails with Clang/libc++ 12:
```c++
src/steamcompmgr.cpp:281:28: error: copying variable of type 'std::atomic<bool>' invokes deleted constructor
static std::atomic< bool > g_bTakeScreenshot = false;
                           ^                   ~~~~~
/usr/include/c++/v1/atomic:1787:7: note: copy constructor of 'atomic<bool>' is implicitly deleted because base class '__atomic_base<bool>' has a deleted copy constructor
    : public __atomic_base<_Tp>
      ^
/usr/include/c++/v1/atomic:1683:5: note: '__atomic_base' has been explicitly marked deleted here
    __atomic_base(const __atomic_base&) = delete;
    ^
```

Reply:
> does g_whatever{true} work?

Yep. `foo{bar}`, `foo(bar)`, `foo = {bar}` compile fine.

```suggestion
static std::atomic< bool > g_bTakeScreenshot{false};
```

Reply:
Thanks, fixed! Also added CI to make sure we catch clang regressions.